imagebuildah/build.go

```diff
@@ -532,10 +532,10 @@ func NewExecutor(store storage.Store, options BuildOptions) (*Executor, error) {
 		annotations:           append([]string{}, options.Annotations...),
 	}
 	if exec.err == nil {
-		exec.err = os.Stderr
+		exec.err = options.ReportWriter
```

Comment:
This will cause a crash in quiet mode, because we previously didn't (and still don't) require `ReportWriter` to be non-`nil`, but depend on `exec.err` being non-`nil` below.